tests/Handler/FpmHandlerTest.php

```diff
@@ -1117,7 +1117,7 @@ private function assertGlobalVariables(array $event, array $expectedGlobalVariab
         // Test global variables that never change (simplifies all the tests)
         $response = $this->assertCommonServerVariables($response, $expectedGlobalVariables);
 
-        self::assertEquals($expectedGlobalVariables, $response);
+        self::assertArraySubset($expectedGlobalVariables, $response);
```

Comment:
Regarding your comment about this: we should aim to keep this as strict as possible. In the past, I went through a lot of hoops to keep this `assertEquals` (as you can see with the weird `assertCommonServerVariables()` above).

The reason is to have 100% confidence about the environment variables available in FPM.

I would be fine with a little hack, e.g. adding `unset()` (to remove the extra key) above with a comment explaining that it's specifically for PHP 8.1 (or anything like that). That would allow to keep the strictness here.

Reply:
I understand, I'll look into it.

Reply:
@mnapoli @deleugpn fixed to tests to keep the assertion strict

Reply:
Thank you!